src/internal/connector/graph/service.go

```diff
@@ -13,6 +13,12 @@ import (
 // given endpoint. The endpoint granularity varies by service.
 const DeltaTokenFileName = "delta"
 
+// MetadataFileNames produces the standard set of filenames used to store graph
+// metadata such as delta tokens and folderID->path references.
+func MetadataFileNames() []string {
```

Comment:
I think this works for now, though in the future we should probably consider more generic use-case like "what if a services has different metadata files/paths for different categories of data"